cegui/include/CEGUI/Window.h

```diff
@@ -658,6 +658,7 @@ class CEGUIEXPORT Window :
     */
     bool isChild(unsigned int ID) const;
 
+    using NamedElement::isChildRecursive;
```

Comment:
One trailing space before `using` though.

Reply:
I counted 4?

Reply:
Oops sorry I was not being thoughtful when commenting that. The plus sign at the beginning made the line unaligned with other code lines so I assumed that there was a space. Checking that with full file view showed correct spacing.